docker/include/network/scripts/start_networking

```diff
@@ -40,8 +40,13 @@ if ! ip addr show dev $LOCAL_IF | fgrep -q 'inet '; then
   ip addr add 10.20.$subnet.1/16 dev $LOCAL_IF
 fi
 
-ip addr add 10.20.255.254/16 dev $LOCAL_IF #For static ip devices' default gateway IP
-echo dhcp-host=*,ignore >> /etc/dnsmasq.conf
+# For static ip devices' default gateway IP
+ip addr add 10.20.255.254/16 dev $LOCAL_IF
+
+# Block execution until first host attaches unless being used as a faux device.
```

Comment:
I think this may be breaking the alt-swtich integration test. The integration tests waits for a log message before setting the dhcp range.

Reply:
I don't think it was this that was breaking the tests, there were other more fundamental problems.  But, either way this is no longer necessary given the new mechanism.  Still running a few final tests but I think I got everything now...

Reply:
Should be fixed now.